definitions/npm/ramda_v0.x.x/flow_v0.82.x-/ramda_v0.x.x.js

```diff
@@ -0,0 +1,2005 @@
+  declare function assocPath<T, S>(
+    key: Array<string>,
+    val: T,
+    ...args: Array<void>
+  ): (src: S) => { [k: string]: T } & S;
+  declare function assocPath<T, S>(
+    key: Array<string>,
+    val: T,
+    src: S
+  ): { [k: string]: T } & S;
+
+  declare function clone<T>(src: T): $Shape<T>;
+
+  declare function dissoc<T>(
+    key: string,
+    ...args: Array<void>
+  ): (src: { [k: string]: T }) => { [k: string]: T };
+  declare function dissoc<T>(
+    key: string,
+    src: { [k: string]: T }
+  ): { [k: string]: T };
+
+  declare function dissocPath<T>(
+    key: Array<string>,
+    ...args: Array<void>
+  ): (src: { [k: string]: T }) => { [k: string]: T };
+  declare function dissocPath<T>(
+    key: Array<string>,
+    src: { [k: string]: T }
+  ): { [k: string]: T };
+
+  declare function evolve<A: Object>(NestedObject<Function>, A): A;
+  declare function evolve<A: Object>(NestedObject<Function>): A => A;
+
+  declare function eqProps(
+    key: string,
+    ...args: Array<void>
+  ): ((o1: Object) => (o2: Object) => boolean) &
+    ((o1: Object, o2: Object) => boolean);
+  declare function eqProps(
+    key: string,
+    o1: Object,
+    ...args: Array<void>
+  ): (o2: Object) => boolean;
+  declare function eqProps(key: string, o1: Object, o2: Object): boolean;
+
+  declare function has(key: string, o: Object): boolean;
+  declare function has(key: string): (o: Object) => boolean;
+
+  declare function hasIn(key: string, o: Object): boolean;
+  declare function hasIn(key: string): (o: Object) => boolean;
+
+  declare function invert(o: Object): { [k: string]: Array<string> };
+  declare function invertObj(o: Object): { [k: string]: string };
+
+  declare function keys(o: ?Object): Array<string>;
+
+  declare type Lens = <T, V>(x: T) => V;
+
+  declare function lens<T, U, V>(
+    getter: (s: T) => U,
+    setter: (a: U, s: T) => V
+  ): Lens;
+  declare function lens<T, U, V>(
+    getter: (s: T) => U
+  ): (setter: (a: U, s: T) => V) => Lens;
+
+  declare function lensIndex(n: number): Lens;
+
+  declare function lensPath(a: Array<string | number>): Lens;
+
+  declare function lensProp(str: string): Lens;
+
+  declare function mapObjIndexed<A, B>(
+    fn: (val: A, key: string, o: Object) => B,
+    o: { [key: string]: A }
+  ): { [key: string]: B };
+  declare function mapObjIndexed<A, B>(
+    fn: (val: A, key: string, o: Object) => B,
+    ...args: Array<void>
+  ): (o: { [key: string]: A }) => { [key: string]: B };
+
+  declare type Merge = (<A, B>(a: A, b: B) => A & B) &
+    (<A, B>(a: A) => (b: B) => A & B);
+
+  declare var merge: Merge;
+
+  declare function mergeAll<T>(
+    os: Array<{ [k: string]: T }>
+  ): { [k: string]: T };
+
+  declare var mergeDeepLeft: Merge;
+
+  declare var mergeDeepRight: (<A, B>(a: A, b: B) => B & A) &
+    (<A, B>(a: A) => (b: B) => B & A);
+
+  declare type MergeWith = (<A: { [k: string]: mixed }, B: { [k: string]: mixed }, T>(
```

Comment:
Hello, @TomPridham ✋ 
This PR seems to resolve type errors the v0.82.0 causing, and I've appreciated such contribution 🙏 
By the way, could we use `typeof T` as annotation instead of `mixed` which weaken strictness of type :question:

Reply:
i couldn't get it to work with `typeof T`. i messed with it for about an hour, but couldn't get anything other than mixed to pass the tests

Reply:
Ah, I see. 
Thanks to explanation 👍